invenio_github/api.py

```diff
@@ -209,11 +209,11 @@ def sync_repo_hook(self, repo_id):
                 pass
 
     def check_sync(self):
-        """Check and sync if over a day has passed since the last sync."""
+        """Check if sync is required based on last sync date."""
         now = utcnow()
-        yesterday = now - timedelta(days=1)
+        expiration = now - current_app.config['GITHUB_REFRESH_TIMEDELTA']
```

Comment:
Can just cover case when `GITHUB_REFRESH_TIMEDELTA is None`?

--
see also https://github.com/inveniosoftware/invenio-oauthclient/blob/master/invenio_oauthclient/contrib/cern.py#L210-L218